src/axom/quest/IntersectionShaper.hpp

```diff
@@ -2906,6 +2990,38 @@ class IntersectionShaper : public Shaper
     return isTri;
   }
 
+  /*!
+   * \brief Returns the dimension of the computational mesh
+   * \return dim The dimensions of the mesh (expected is 2 or 3, -1 in case of failure)
+   */
+  int getCompMeshDim()
```

Comment:
Do the objects used in here permit this method to be marked ``const``?

Reply:
Gave this a shot, but there are mfem API function calls that are not currently `const` (the first error I received was related to `GetMesh()`, but there are possibly more).